sos/report/plugins/salt.py

```diff
@@ -6,17 +6,17 @@
 #
 # See the LICENSE file in the source distribution for further information.
 
-from sos.report.plugins import Plugin, RedHatPlugin, DebianPlugin
+from sos.report.plugins import Plugin, IndependentPlugin
 
 
-class Salt(Plugin, RedHatPlugin, DebianPlugin):
+class Salt(Plugin, IndependentPlugin):
 
     short_desc = 'Salt'
 
     plugin_name = 'salt'
     profiles = ('sysmgmt',)
 
-    packages = ('salt',)
+    packages = ('salt-minion', 'salt-common',)
```

Comment:
There is a package named `salt` on RHEL distributions, in addition to `salt-minion` (but there is no `salt-common`). I'd prefer we leave the original in place as well given this.

Reply:
Done.